google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -68,15 +68,6 @@ def classify_stmt(sql):
 )
 
 
-def strip_backticks(name):
-    """
-    Strip backticks off of quoted names For example, '`no`' (a Spanner reserved
-    word) becomes 'no'.
-    """
-    has_quotes = name.startswith("`") and name.endswith("`")
-    return name[1:-1] if has_quotes else name
```

Comment:
Doesn't look like we actually need such a logic. This function is used only on column names. Column names themselves can not start or end with backticks, so backticks are used only as a marking. Means we can just `strip()` both whitespaces and backticks with one method call.